app/main.py

```diff
@@ -1,8 +1,54 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            comfort_class: int,
+            clean_mark: int,
+            brand: str
+    ) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
```

Comment:
remove, please, "#write your code here" and "pass" keyword

Reply:
Same here [screen from IDE](![Screenshot 2023-02-06 at 22 56 52](https://user-images.githubusercontent.com/122725491/217084716-4ab984da-5417-4114-8124-fabfeb75c1a6.png))

Reply:
Ostap, when you open "Files changed" tab:

The RED lighted strings mean, that this strings were DELETED by me.   
The GREEN lighted strings mean, that this strings were WRITTEN by me.

You can press <Settings button> and change Diff view for better understanding 
![Screenshot 2023-02-06 at 23 27 25](https://user-images.githubusercontent.com/122725491/217093086-47c92f60-0959-4913-9bde-3792020c751e.png)